diff --git a/dist/ControlsPanel.js b/dist/ControlsPanel.js
index 98656a12cdb8afefbee46c8cdeb8e27205316d10..08358cd48cbefca8a6bc5469f4750ca0de759001 100644
--- a/dist/ControlsPanel.js
+++ b/dist/ControlsPanel.js
@@ -1,9 +1,9 @@
 import { jsx as _jsx } from "react/jsx-runtime";
 import { includeConditionalArg, } from 'storybook/internal/csf';
 import React, { useCallback, useState } from 'react';
-import NoControlsWarning from './NoControlsWarning';
-import PropForm from './PropForm';
-import { useArgs } from './hooks';
+import NoControlsWarning from './NoControlsWarning.js';
+import PropForm from './PropForm.js';
+import { useArgs } from './hooks.js';
 function shouldIncludeArg(argType, args) {
     try {
         return includeConditionalArg(argType, args, {});
diff --git a/dist/PropField.js b/dist/PropField.js
index f5247c7674cb2e788daf47b343bc6f5708715024..1a9c0961c21a514b8f592b1bcca795f2f63231bc 100644
--- a/dist/PropField.js
+++ b/dist/PropField.js
@@ -1,6 +1,6 @@
 import { jsxs as _jsxs, jsx as _jsx } from "react/jsx-runtime";
 import { styled } from '@storybook/react-native-theming';
-import TypeMap from './types';
+import TypeMap from './types/index.js';
 const InvalidTypeText = styled.Text(({ theme }) => ({
     color: theme.color.negative,
 }));
diff --git a/dist/PropForm.js b/dist/PropForm.js
index 984d158440e9d4cf775d62447efead408f0b587b..80f4e3bb3a079d7ac1d08d0076c00d198a56bec2 100644
--- a/dist/PropForm.js
+++ b/dist/PropForm.js
@@ -1,7 +1,7 @@
 import { jsx as _jsx, jsxs as _jsxs } from "react/jsx-runtime";
 import { styled } from '@storybook/react-native-theming';
 import { transparentize } from 'polished';
-import PropField from './PropField';
+import PropField from './PropField.js';
 import { Image, TouchableOpacity, View } from 'react-native';
 const Container = styled.View(() => ({ paddingTop: 0 }));
 const Label = styled.Text(({ theme }) => ({
diff --git a/dist/components/SelectModal.js b/dist/components/SelectModal.js
index 7a5df34a343506c41010651c4900a4580745464a..3df5a10f4dad33c8ddc651e9aec0049dc6c52b8c 100644
--- a/dist/components/SelectModal.js
+++ b/dist/components/SelectModal.js
@@ -2,7 +2,7 @@ import { jsx as _jsx, jsxs as _jsxs } from "react/jsx-runtime";
 import { styled } from '@storybook/react-native-theming';
 import { useCallback, useEffect, useMemo, useState } from 'react';
 import { ScrollView, StyleSheet, TouchableOpacity, TouchableWithoutFeedback, useWindowDimensions, View, } from 'react-native';
-import { ModalPortal } from './ModalPortal';
+import { ModalPortal } from './ModalPortal.js';
 const styles = StyleSheet.create({
     overlay: {
         flex: 1,
diff --git a/dist/components/SliderWrapper.js b/dist/components/SliderWrapper.js
index 1294d133fb0bec6d1ef783cb9cb161a0d39b57a3..456ff6119dae284ed11d8150ea96df9fd8aa5c03 100644
--- a/dist/components/SliderWrapper.js
+++ b/dist/components/SliderWrapper.js
@@ -1,5 +1,5 @@
 import { jsx as _jsx } from "react/jsx-runtime";
-import CustomSlider from './Slider';
+import CustomSlider from './Slider.js';
 import { Platform } from 'react-native';
 let Slider = null;
 try {
diff --git a/dist/components/color-picker/HoloColorPicker.js b/dist/components/color-picker/HoloColorPicker.js
index 75d9b29f9b3cfc68ddf3ed005ec75d2d5568481e..6a18cf41685dc54d30c3b2d3707c81b124e6a420 100644
--- a/dist/components/color-picker/HoloColorPicker.js
+++ b/dist/components/color-picker/HoloColorPicker.js
@@ -2,8 +2,8 @@ import { jsx as _jsx, jsxs as _jsxs, Fragment as _Fragment } from "react/jsx-run
 import { useCallback, useEffect, useMemo, useRef, useState } from 'react';
 import { I18nManager, Image, StyleSheet, Text, View, } from 'react-native';
 import tinycolor from 'tinycolor2';
-import SliderWrapper from '../SliderWrapper';
-import { createPanResponder } from './utils';
+import SliderWrapper from '../SliderWrapper.js';
+import { createPanResponder } from './utils.js';
 const getHsv = (color) => tinycolor(color).toHsv();
 export function HoloColorPicker({ defaultColor, oldColor, onColorChange, style, }) {
     const [color, setColor] = useState(() => getHsv(defaultColor));
diff --git a/dist/components/color-picker/index.js b/dist/components/color-picker/index.js
index 39007b46f5c27c28c576755111ffcb057a55eee4..3509132b092a3c97d32b1db4177ecbab82fc1ca6 100644
--- a/dist/components/color-picker/index.js
+++ b/dist/components/color-picker/index.js
@@ -1,2 +1,2 @@
-export { fromHsv, toHsv } from './utils';
-export { HoloColorPicker as ColorPicker } from './HoloColorPicker';
+export { fromHsv, toHsv } from './utils.js';
+export { HoloColorPicker as ColorPicker } from './HoloColorPicker.js';
diff --git a/dist/index.js b/dist/index.js
index 63fa6280422bea6df8274d29c235472fd1258c56..7012e15c34be3fbbf846d4cfeea1e7f391b033c2 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -1,7 +1,7 @@
 import { jsx as _jsx } from "react/jsx-runtime";
 import { addons, types } from 'storybook/manager-api';
-import ControlsPanel from './ControlsPanel';
-import { AddonPanel } from './Panel';
+import ControlsPanel from './ControlsPanel.js';
+import { AddonPanel } from './Panel.js';
 export const ADDON_ID = 'RNCONTROLS';
 export const PARAM_KEY = 'controls';
 export function register() {
diff --git a/dist/types/Array.js b/dist/types/Array.js
index eefc9d1ae0e4fc3bfd57c088220ad2f303c40abf..49d244a90a7e547a23f3484afa8744d32d36a1eb 100644
--- a/dist/types/Array.js
+++ b/dist/types/Array.js
@@ -1,7 +1,7 @@
 import { jsx as _jsx } from "react/jsx-runtime";
 import { useState } from 'react';
-import { useResyncValue } from './useResyncValue';
-import { Input } from './common';
+import { useResyncValue } from './useResyncValue.js';
+import { Input } from './common.js';
 function formatArray(value, separator) {
     if (value === '') {
         return [];
diff --git a/dist/types/Color.js b/dist/types/Color.js
index 7f76542c5321523ec2e2fb3aaa7aa83b316000d1..942033b8e9bfadc897060b5571922c6791ddec2e 100644
--- a/dist/types/Color.js
+++ b/dist/types/Color.js
@@ -2,8 +2,8 @@ import { jsx as _jsx, jsxs as _jsxs } from "react/jsx-runtime";
 import { useState } from 'react';
 import { View, TouchableWithoutFeedback, StyleSheet, Platform, Dimensions } from 'react-native';
 import { styled, useTheme } from '@storybook/react-native-theming';
-import { ColorPicker, fromHsv, toHsv } from '../components/color-picker';
-import { ModalPortal } from '../components/ModalPortal';
+import { ColorPicker, fromHsv, toHsv } from '../components/color-picker/index.js';
+import { ModalPortal } from '../components/ModalPortal.js';
 const TouchableContainer = styled.TouchableOpacity(({ theme }) => ({
     width: 44,
     height: 44,
diff --git a/dist/types/Date.js b/dist/types/Date.js
index efb49993d4f8720c36c2aa6fa02f20440da672dc..035a911d961b509f0936e988f57d87870010decc 100644
--- a/dist/types/Date.js
+++ b/dist/types/Date.js
@@ -3,7 +3,7 @@ import { styled, useTheme } from '@storybook/react-native-theming';
 import { useMemo, useState } from 'react';
 import { Platform, StyleSheet, View } from 'react-native';
 import DateTimePicker from 'react-native-modal-datetime-picker';
-import { inputStyle } from './common';
+import { inputStyle } from './common.js';
 const Touchable = styled.TouchableOpacity(({ theme }) => ({
     ...inputStyle({ theme }),
 }));
diff --git a/dist/types/Number.js b/dist/types/Number.js
index 8e8061fcb4c144240f0e152dac5a8fb605b8647f..7d676ea11fad05d55cb5d2467f58a60f8963a353 100644
--- a/dist/types/Number.js
+++ b/dist/types/Number.js
@@ -2,9 +2,9 @@ import { jsx as _jsx, jsxs as _jsxs } from "react/jsx-runtime";
 import { styled } from '@storybook/react-native-theming';
 import { useCallback, useState } from 'react';
 import { View } from 'react-native';
-import { useResyncValue } from './useResyncValue';
-import { Input } from './common';
-import SliderWrapper from '../components/SliderWrapper';
+import { useResyncValue } from './useResyncValue.js';
+import { Input } from './common.js';
+import SliderWrapper from '../components/SliderWrapper.js';
 const ValueContainer = styled.View({ flexDirection: 'row' });
 const LabelText = styled.Text(({ theme }) => ({
     color: theme.color.mediumdark,
diff --git a/dist/types/Object.js b/dist/types/Object.js
index e09140d10c96788a1d68aace316c86f64abe3786..9f855965485f662e2ed4bd9f9175e724f6a9bced 100644
--- a/dist/types/Object.js
+++ b/dist/types/Object.js
@@ -1,7 +1,7 @@
 import { jsx as _jsx } from "react/jsx-runtime";
 import { useCallback, useState } from 'react';
-import { Input } from './common';
-import { useResyncValue } from './useResyncValue';
+import { Input } from './common.js';
+import { useResyncValue } from './useResyncValue.js';
 const ObjectType = ({ arg, onChange, isPristine }) => {
     const getJsonString = useCallback(() => {
         try {
diff --git a/dist/types/Radio.js b/dist/types/Radio.js
index 8b1ef777a479008416a48c62fdc33a68b9097938..2a70d0f028523f28480e6bcd165915d91fe81276 100644
--- a/dist/types/Radio.js
+++ b/dist/types/Radio.js
@@ -1,6 +1,6 @@
 import { jsx as _jsx } from "react/jsx-runtime";
 import React from 'react';
-import RadioSelect from '../components/RadioSelect';
+import RadioSelect from '../components/RadioSelect.js';
 const getOptions = (arg) => {
     if (arg.options) {
         return arg.options;
diff --git a/dist/types/Select.js b/dist/types/Select.js
index 44ef0b7e67b2911b839b116b1f71bec82becd9d9..72e910fc6ba2c764505b3a315f604a2005570f17 100644
--- a/dist/types/Select.js
+++ b/dist/types/Select.js
@@ -2,8 +2,8 @@ import { jsx as _jsx } from "react/jsx-runtime";
 import { useTheme } from '@storybook/react-native-theming';
 import { useMemo } from 'react';
 import { Platform, View } from 'react-native';
-import { SelectModal } from '../components/SelectModal';
-import { Input, inputStyle } from './common';
+import { SelectModal } from '../components/SelectModal.js';
+import { Input, inputStyle } from './common.js';
 const getOptions = (options, labels) => {
     if (Array.isArray(options)) {
         return options.map((val) => ({ key: val, label: String(labels?.[val] || val) }));
diff --git a/dist/types/Text.js b/dist/types/Text.js
index cfeea30f0df4e6d2b8ab1fbd8cc46a09f31812e1..2a74b7f6e271123a7166fee6ae67b1b080b99abb 100644
--- a/dist/types/Text.js
+++ b/dist/types/Text.js
@@ -1,7 +1,7 @@
 import { jsx as _jsx } from "react/jsx-runtime";
 import { useState } from 'react';
-import { Input } from './common';
-import { useResyncValue } from './useResyncValue';
+import { Input } from './common.js';
+import { useResyncValue } from './useResyncValue.js';
 const TextType = ({ arg, onChange, isPristine }) => {
     const { setCurrentValue, key } = useResyncValue(arg.value, isPristine);
     const [focused, setFocused] = useState(false);
diff --git a/dist/types/index.js b/dist/types/index.js
index 3b7015b90f6a89933d3c2629bba30135408941be..a8e3dc24bcdbe504fef45d3600fef8a3dffab102 100644
--- a/dist/types/index.js
+++ b/dist/types/index.js
@@ -1,12 +1,12 @@
-import TextType from './Text';
-import NumberType from './Number';
-import ColorType from './Color';
-import BooleanType from './Boolean';
-import ObjectType from './Object';
-import SelectType from './Select';
-import DateType from './Date';
-import ArrayType from './Array';
-import RadioType from './Radio';
+import TextType from './Text.js';
+import NumberType from './Number.js';
+import ColorType from './Color.js';
+import BooleanType from './Boolean.js';
+import ObjectType from './Object.js';
+import SelectType from './Select.js';
+import DateType from './Date.js';
+import ArrayType from './Array.js';
+import RadioType from './Radio.js';
 export default {
     text: TextType,
     number: NumberType,
