diff --git a/plugin/index.js b/plugin/index.js
index 6419dc44d51d8e46743c1f69b84bfc4771802dec..5ce424cd96ec3ea01d050c2cadeeb623a8e3d0d0 100644
--- a/plugin/index.js
+++ b/plugin/index.js
@@ -694,7 +694,7 @@ var require_workletStringCode = __commonJS({
       if (includeSourceMap) {
         inputMap.sourcesContent = [];
         for (const sourceFile of inputMap.sources) {
-          inputMap.sourcesContent.push(fs.readFileSync(sourceFile).toString("utf-8"));
+          inputMap.sourcesContent.push(fs.readFileSync(sourceFile.replace(/[?#].*$/, "")).toString("utf-8"));
         }
       }
       const transformed = (0, transform_1.workletTransformSync)(code, {
